velox/exec/Aggregate.h

```diff
@@ -313,6 +314,52 @@ class Aggregate {
   std::vector<vector_size_t> pushdownCustomIndices_;
 };
 
+/**
+ *  AverageDecimalAccumulator appends an additional member, overflow count
+ *  to SumCount base class:
+ *    SUM: Total sum.
+ *    COUNT: Total number of rows so far.
+ *    OVERFLOW: Number of times the sum overflow/underflow occurred.
+ */
+struct AverageDecimalAccumulator {
```

Comment:
Wrong file

Reply:
I need this in a header to write unit tests. Do you have any other suggestions?

Reply:
You should put it in its own header file like `velox/functions/prestosql/aggregates/AverageAggregate.h`